x-pack/plugins/ml/common/types/storage.ts

```diff
@@ -34,6 +36,9 @@ export type PartitionFieldsConfig =
   | Partial<Record<EntityFieldType, PartitionFieldConfig>>
   | undefined;
 
+export type ApplyTimeRangeConfig = boolean;
```

Comment:
should it also contain `undefined`?
```suggestion
export type ApplyTimeRangeConfig = boolean | undefined;
```

Reply:
Updated here [`82b3c33` (#97407)](https://github.com/elastic/kibana/pull/97407/commits/82b3c330d69d78c0386ca7164db80736b36ffe0b)